### PR TITLE
Autosize inputs

### PR DESCRIPTION
**Pull request by YaroslavNikiforov**

### CODE REVIEW

File: front-end/src/components/Admin/Record/RecordEditor/index.js [hunk 1]
```diff
@@ -100,24 +93,18 @@ class RecordEditor extends Component {
...
           formData={getRecordData()}
           autocomplete='off'
           onChange={this.changeData}
           onSubmit={this.saveData}
-          onError={this.log('errors')}
-        >
-          <button className='record__button'>Сохранить</button>
-        </Form>}
+          onError={this.log('errors')}>
+          <button ref={saveButton => (this.saveButton = saveButton)} onClick={this.saveData} className='record__button'>Сохранить</button>
```
Comment (Thealoner) [inline front-end/src/components/Admin/Record/RecordEditor/index.js]: Here is an example of how you control an element state (enabled/disabled). It has to be based on component state. We don't need refs here.
```
<button disabled={this.state.loading} onClick={this.saveData} className='record__button'>Сохранить</button>
```

File: front-end/src/components/Admin/Record/RecordEditor/index.js [hunk 2]
```diff
@@ -5,34 +5,33 @@ import 'tabulator-tables/dist/css/tabulator.min.css';
...
     };
   }
 
   getData = () => {
     let { rowId } = this.props.match.params;
     let { entityType } = this.props;
     let entity = getEntityByType(entityType);
-
-    this.setState({
-      loading: true
```
[the remark this answers is not in the document]
Comment (YaroslavNikiforov) [reply front-end/src/components/Admin/Record/RecordEditor/index.js]: I think that the response from server comes so quickly that we don't need to show the preloader flickering

File: front-end/src/components/Admin/Record/RecordEditor/index.js [hunk 3]
```diff
@@ -41,24 +40,17 @@ class RecordEditor extends Component {
     let { entityType, setTabContentUrl } = this.props;
     let entity = getEntityByType(entityType);
 
-    this.setState({
-      loading: true
-    });
+    this.saveButton.disabled = true;
```
[the remark this answers is not in the document]
Comment (YaroslavNikiforov) [reply front-end/src/components/Admin/Record/RecordEditor/index.js]: absolutely agree

File: front-end/src/components/Admin/Record/RecordEditor/index.js (hunk 1 above)
Comment (YaroslavNikiforov) [reply front-end/src/components/Admin/Record/RecordEditor/index.js]: that's right )

**Pull request merged**